src/index.ts

```diff
@@ -26,81 +27,105 @@ export const languages = ["en", "fr", "es", "ru", "cs", "uk", "pt", "sk"];
  * locale.draw.toolbar.buttons.polygon = 'Awesome polygon!'
  * L.drawLocal = locale
  */
-export default function(language: string): DrawLocal {
+export const drawLocales = (language: Language): DrawLocal => {
   let locale = en;
   switch (language) {
     case "en":
-    case "en_US":
```

Comment:
going to modify these changes

It's actually supposed to be `en-US`

Based on the text strings provided by `navigator.languages`

![image](https://user-images.githubusercontent.com/550895/62409717-c63f1900-b5db-11e9-8a58-a9a06182fc5d.png)

Reply:
Or we set `language.toLocaleLowerCase()`

Reply:
Oh, I didn't know it matches on `navigator.languages`, I'll revert some changes and create a new PR

Reply:
Oh I see you already added a commit to cover this? So no more changes needed?

I did lowercase because it makes for easier matching and I thought it was only on what you give in the `drawLocales` function, but you seem to give me the impression that leaflet-draw calls t he `drawLocales` function  interanally with `navigator.language` as well?